builder/proxmox/step_start_vm.go

```diff
@@ -118,6 +119,15 @@ func generateProxmoxDisks(disks []diskConfig) proxmox.QemuDevices {
 	}
 	return devs
 }
+func generateProxmoxVga(vga vgaConfig) proxmox.QemuDevice {
+	dev := make(proxmox.QemuDevice)
+	setDeviceParamIfDefined(dev, "type", vga.Type)
+
+	if vga.Memory > 0 {
```

Comment:
What happens if this is set, but `vga.Type` is not? If it causes parameter validation errors in Proxmox, we should add validation in `config.Prepare`

Reply:
It appears you can set either or without ill effect. The `memory` directive will be applied to the default adapter. The `qm.conf` comes out looking like this:

```
#Packer ephemeral build VM
agent: 1
...
vga: memory=48
...
```
![Screenshot from 2020-03-16 11-46-26](https://user-images.githubusercontent.com/669283/76804748-4cd6ee00-679a-11ea-947f-b890bc2514c7.png)

I will remove the `Required` note [here](https://github.com/hashicorp/packer/pull/8892/files#diff-4d8ad6a99d84244b1840b1c869d3edd4R107) and repush @carlpett .